Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM pytorch/pytorch:2.5.1-cuda12.4-cudnn9-runtime
```

Comment:
@lewtun Do we need to switch to `12.1` for https://github.com/huggingface/open-r1/pull/121 ?

Reply:
I'd think so, did you manage to actually reproduce anything on the 14.1? I looked at it and really didn't see how it would actually run, when I ran on 12.4 it gave a segmentation fault.